lib/modules/manager/kubernetes/extract.ts

```diff
@@ -73,6 +79,7 @@ function extractApis(content: string, fileName: string): PackageDependency[] {
         is.nonEmptyStringAndNotWhitespace(m.kind) &&
         is.nonEmptyStringAndNotWhitespace(m.apiVersion)
     )
+    .filter((m) => Object.keys(kubernetesApiVersions).includes(m.kind))
```

Comment:
Maybe push this Object.keys() up into the top scope so that it's not recalculated every time?

Reply:
yes, should be stored as Set for perf